geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/parallel/ParallelWANPersistenceEnabledGatewaySenderDUnitTest.java

```diff
@@ -1892,9 +1908,10 @@ public void testpersistentWanGateway_restartSender_expectAllEventsReceived_scena
     LogWriterUtils.getLogWriter().info("All the senders are now running...");
 
     // ----------------------------------------------------------------------------------------------------
-
-    vm2.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName(), 3000));
-    vm3.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName(), 3000));
+    vm7.invoke(() -> WANTestBase.doPuts(getTestMethodName(), 500));
+    vm4.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName(), 3000));
+    vm2.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName(), 500));
+    vm3.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName(), 500));
```

Comment:
I do not understand why this case should behave differently now. I would expect that the legacy behavior is kept.
Why is it?